actions/upgrades

```diff
@@ -0,0 +1,123 @@
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+Configures or runs unattended-upgrades
+"""
+
+import argparse
+import os
+import subprocess
+
+CONF_FILE = '/etc/apt/apt.conf.d/50unattended-upgrades'
+AUTO_CONF_FILE = '/etc/apt/apt.conf.d/20auto-upgrades'
+
+
+def parse_arguments():
+    """Return parsed command line arguments as dictionary"""
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(dest='subcommand', help='Sub command')
+
+    # Run unattended-upgrades
+    subparsers.add_parser('run', help='Upgrade packages on the system')
+
+    # Check if automatic upgrades are enabled
+    subparsers.add_parser('check-auto',
+                          help='Check if automatic upgrades are enabled')
+
+    # Enable automatic upgrades
+    subparsers.add_parser('enable-auto', help='Enable automatic upgrades')
+
+    # Disable automatic upgrades
+    subparsers.add_parser('disable-auto', help='Disable automatic upgrades.')
+
+    return parser.parse_args()
+
+
+def subcommand_run(_):
+    """Run unattended-upgrades"""
+    try:
+        setup()
+    except FileNotFoundError:
+        print('Error: Could not configure unattended-upgrades.')
+        return
+
+    try:
+        output = subprocess.check_output(['unattended-upgrades', '-v'])
+    except subprocess.CalledProcessError as error:
+        print('Error: %s', error)
+    except FileNotFoundError:
+        print('Error: unattended-upgrades is not available.')
+    else:
+        print('%s', output.decode())
+
+
+def subcommand_check_auto(_):
+    """Check if automatic upgrades are enabled"""
+    print(os.path.isfile(AUTO_CONF_FILE))
+
+
+def subcommand_enable_auto(_):
+    """Enable automatic upgrades"""
+    try:
+        setup()
+    except FileNotFoundError:
+        print('Error: Could not configure unattended-upgrades.')
+        return
+
+    with open(AUTO_CONF_FILE, 'w') as conffile:
+        conffile.write('APT::Periodic::Update-Package-Lists "1";\n')
+        conffile.write('APT::Periodic::Unattended-Upgrade "1";\n')
+
+
+def subcommand_disable_auto(_):
+    """Disable automatic upgrades"""
+    try:
+        os.rename(AUTO_CONF_FILE, AUTO_CONF_FILE + '.disabled')
+    except FileNotFoundError:
+        print('Already disabled.')
+
+
+def setup():
+    """Sets unattended-upgrades config to upgrade any package from Debian."""
+    with open(CONF_FILE, 'r') as conffile:
+        lines = conffile.readlines()
+
+    for line in lines:
+        if '"origin=Debian";' in line:
+            return  # already configured
+
+    with open(CONF_FILE, 'w') as conffile:
+        for line in lines:
+            conffile.write(line)
+            if 'Unattended-Upgrade::Origins-Pattern {' in line:
```

Comment:
Not too important, but what if this part is changed to

    Unattended-Upgrade::Origins-Pattern
    {
        ...

Would that be understood by the parser too? Or is the format we look for a debian convention (or like that for years already)?